CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ since 0.4.1, and this project adheres to [Semantic Versioning](https://semver.or
 
 ## Fixed
 - Fix mismatched between between original document and converted one ([issue #626](https://github.com/freedomofpress/dangerzone/issues/)). This does not affect the quality of the final document.
+- Capitalize "dangerzone" on the application as well as on the Linux desktop shortcut.
```

Comment:
Let's add a thanks to the original contributor here! I also see that you have retained the original contributor as a co-author, thanks! Other than that, feel free to merge.